components/KiboPayment.vue

```diff
@@ -0,0 +1,200 @@
+      >
+        <template #label>
+          <div class="sf-radio__label">
+            {{ item.label }}
+          </div>
+        </template>
+      </SfRadio>
+    </template>
+
+    <template #credit-card-form>
+      <div v-if="paymentMethod" class="credit-card-form">
+        <div class="credit-card-form__element custom-form-element sf-input">
+          <div class="sf-input__wrapper">
+            <input
+              v-model="creditCardFormData.cardNumber"
+              type="number"
+              name="cardNumber"
+              class="custom-form-element__input"
+              :required="false"
+              @input="updateField()"
+            />
+            <SfButton class="custom-form-element__button sf-button--pure">
+              <span class="custom-form-element__icon">
+                <SfIcon size="1.25rem">
+                  <font-awesome-icon icon="credit-card" class="fa-icon" />
+                </SfIcon>
+              </span>
+            </SfButton>
+            <label for="cardNumber" class="sf-input__label">{{ $t("Card number") }}</label>
+          </div>
+          <div class="sf-input__error-message"><div class="display-none"></div></div>
+        </div>
+        <SfInput
+          v-model="creditCardFormData.expiryDate"
+          :label="$t('Exp. Date MM/YY')"
+          name="expiryDate"
+          class="credit-card-form__element"
+          :required="true"
+          @input="updateField()"
+        />
+        <div class="credit-card-form__element custom-form-element sf-input">
+          <div class="sf-input__wrapper">
+            <input
+              v-model="creditCardFormData.securityCode"
+              type="password"
+              name="securityCode"
+              class="custom-form-element__input"
+              :required="true"
+              @input="updateField()"
+            />
+            <SfButton class="custom-form-element__button sf-button--pure">
+              <span class="custom-form-element__icon">
+                <SfIcon size="1.25rem">
+                  <font-awesome-icon icon="question-circle" class="fa-icon" />
+                </SfIcon>
+              </span>
+            </SfButton>
+            <label for="securityCode" class="sf-input__label">{{ $t("Security Code") }}</label>
+          </div>
+          <div class="sf-input__error-message"><div class="display-none"></div></div>
+        </div>
+        <SfCheckbox
+          v-model="creditCardFormData.cardKeep"
+          name="keepcard"
+          :disabled="!paymentMethod"
+          label="Save payment method"
+          class="credit-card-form__checkbox"
+          @change="updateField()"
+        />
+      </div>
+    </template>
+  </SfPayment>
+</template>
+<script lang="ts">
+import { defineComponent } from "@vue/composition-api"
+import { SfPayment, SfRadio, SfInput, SfCheckbox, SfIcon, SfButton } from "@storefront-ui/vue"
+import { ref } from "@nuxt/bridge/dist/runtime"
```

Comment:
This should come from @nuxtjs/composition-api ?

Reply:
Done